CHANGES.rst

```diff
@@ -11,6 +11,11 @@
 
 - 100% branch coverage.
 
+- Make the C and Python `TimeStamp` objects behave more alike. The
+  Python version now produces the same `repr` and `.raw()` output as
+  the C version, and has the same hashcode. In addition, the Python
+  version is now supports ordering and equality like the C version.
+
```

Comment:
Nitpick: I believe the convention is two blank lines between sections.

(It probably doesn't matter.  Why am I even commenting?)